test/powershell/Language/Scripting/Redirection.Tests.ps1

```diff
@@ -0,0 +1,10 @@
+
+Describe "File redirection" -Tag CI {
```

Comment:
we already have redirection tests here: test\powershell\Language\Parser\RedirectionOperator.Tests.ps1
do these really need to go into a new file?